.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This is a bad change. Please add `.vscode` to our `.gitignore` and delete the file

Btw, I think this config isn't helpful to begin with. The Sphinx confDir is in `example_docs`, not `.` (the repository "root").